mitodl/bootcamp-ecommerce
# Update profile page styling/layout

Based on and blocked by #537

#### What are the relevant tickets?
Closes #547

#### What's this PR do?
Restyles view/edit profile pages to drawer content

#### How should this be manually tested?
- Go to `../drawer` - this is a temporary page for testing that should eventually be deleted.
- Click "View Profile' or 'Edit Profile' - verify that the correct content shows up in the drawer.  Click outside the drawer to close it.
- On the 'View Profile` display click the 'Edit' button.  The 'Edit Profile' form should show up in the drawer.  Make changes and save.  The 'View Profile' display should show up again with your changes.

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
<img width="599" alt="Screen Shot 2020-06-01 at 9 39 31 AM" src="https://user-images.githubusercontent.com/187676/83421706-85c84b80-a3f6-11ea-9c8b-3cf95c346b2a.png">

<img width="598" alt="Screen Shot 2020-06-01 at 9 39 47 AM" src="https://user-images.githubusercontent.com/187676/83421721-895bd280-a3f6-11ea-89a1-3f53d4cfea65.png">
